proposals/2019/Move Plants from HVAC Systems to Facilities.md

```diff
@@ -0,0 +1,58 @@
+# Move Plants from HVAC Systems to Facilities
+
+## Overview
+
+This proposal is to move the `<auc:Plants>` element from the `<auc:HVACSystemType>` element to the `<auc:Facility>` element.
+
+## Justification
+
+The justification is three-fold:
+
+1. Plants cannot be asserted without first asserting the HVAC system, i.e., the `<auc:Plants>` element can only be included in an XML document if it is a child element of an `<auc:HVACSystemType>` element.
```

Comment:
@jasondegraw What if a user simply wants to assert that there exists a facility and that it contains a plant? This is the case for LL87, where plants may be shared between two NYC properties, "A" and "B", for example. In this case, the user may not know which HVAC system in "A" the plant belongs to; however, they know which HVAC system in "B" is sharing the plant. Hence, there would be no `<HVACSystem>` element in "A", but there would be one in "B".

Reply:
@markborkum As I said, the plants themselves are HVAC systems. There are certainly plant types that wouldn't be HVAC systems, but that's not the case here. Here's an example:

```xml
<BuildingSync>
  <Facilities>
    <Facility>
      <Systems>
        <HVACSystems>
          <HVACSystem ID="BoilerSystem">
            <Plants>
              <HeatingPlants>
                <HeatingPlant ID="Boiler">
                  <Boiler>
                    <BoilerType>Hot water</BoilerType>
                    <DraftType>Mechanical forced</DraftType>
                  </Boiler>
                </HeatingPlant>
              </HeatingPlants>
            </Plants>
          </HVACSystem>
        </HVACSystems>
      </Systems>
    </Facility>
  </Facilities>
</BuildingSync>
```

I can't validate that snippet right now, hopefully the intent comes through. There're certainly options as to how the data gets represented in the various sharing situations, but there's no ambiguity about a boiler being considered an HVAC system. Regarding your "A" and "B" example, there's simply no ambiguity there. The user doesn't need to know which system the plant "belongs to" because the plant is an HVAC system in its own right. It doesn't necessarily follow that there are one or none or any other number of `<HVACSystem>' tags in "A" or "B".